### DISCUSSION TITLE
Laravel Lang

### DISCUSSION THREAD

**Discussion by andrey-helldar** [HassanZahirnia/laravel-package-ocean#11] [Package Suggestions]
List of 78 languages for Laravel Framework, Laravel Jetstream, Laravel Fortify, Laravel Breeze, Laravel Cashier, Laravel Nova, Laravel Spark and Laravel UI.

https://laravel-lang.com

**Comment by pxlrbt** [comment]
I think this is a essential package for every non-english Laravel project. I install it on every project.

**Reply by andrey-helldar** [reply]
Yes. English-only applications do not need this package.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊, I've added the package! It will be live in a moment.
